spanner/api/Spanner.Samples/UpdateUsingDmlReturningAsyncPostgre.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_postgresql_update_dml_returning]
+
+using Google.Cloud.Spanner.Data;
+using Google.Cloud.Spanner.V1;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+public class UpdateUsingDmlReturningAsyncPostgreSample
+{
+
```

Comment:
nit: Unnecessary blank line. Please also see if the above comments are applicable here.